lib/register/template.js

```diff
@@ -32,11 +33,11 @@ var fs = require('fs'),
  */
 function registerTemplate(options) {
   if (typeof options.name !== 'string')
-    throw new Error('transform name must be a string');
+    throw new Error('Transform name must be a string: ' + chalk.red(JSON.stringify(options.name)));
```

Comment:
One small thing: can you change 'Transform' to 'Template' here? Obviously not your fault, this was a poor copy+paste job by myself a long time ago. 😬

Reply:
Done.